pennylane/ops/op_math/controlled.py

```diff
@@ -338,12 +339,14 @@ def _try_wrap_in_custom_ctrl_op(op, control, control_values=None, work_wires=Non
         return ops_with_custom_ctrl_ops[custom_key](*op.data, control + op.wires)
 
     if isinstance(op, qml.QubitUnitary):
-        return qml.ControlledQubitUnitary(
-            op.matrix(),
-            wires=control + op.wires,
+        new_op = qml.ops.ControlledOp(
```

Comment:
```suggestion
        new_op = ControlledOp(
```
Lol I forgot `ControlledOp` is right here